action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'cibuildwheel'
+description: 'Installs and runs cibuildwheel on the current runner'
+inputs:
+  package-dir:
+    description: 'Input directory, defaults to "."'
+    required: false
+    default: '.'
+  output-dir:
+    description: 'Folder to place the outputs in, defaults to "wheelhouse"'
+    required: false
+    default: 'wheelhouse'
+
+runs:
+  using: "composite"
+  steps:
+
+    # This needs powershell-core because github.action path is a Windows-style
+    # path on Windows.  Powershell-core understands both types of paths.
+    - run: python -m pip install ${{ github.action_path }}
+      shell: pwsh
+
+    - run: python -m cibuildwheel ${{ inputs.package-dir }} --output-dir ${{ inputs.output-dir }}
+      shell: bash
```

Comment:
```suggestion
      shell: bash

branding:
  icon: check-circle
  color: red
```

Reply:
There is a package: https://haya14busa.github.io/github-action-brandings/

Where does this show up?

Reply:
This is what I see without the change:
 
<img width="810" alt="Screen Shot 2020-12-23 at 4 22 48 PM" src="https://user-images.githubusercontent.com/4616906/103037970-26b95800-453b-11eb-82eb-9132f9f7d500.png">

Will this turn that red?

Reply:
no, it changes how it appears in the Github Actions Marketplace. e.g. I have an action with [this action.yml](https://github.com/joerick/pr-labels-action/blob/main/action.yml), and it appears on marketplace [like this](https://github.com/marketplace/actions/pr-labels)

Reply:
Ahh, okay. Do you like the check-circle or do you want the package (or anything else)?

Reply:
I'm actually thinking yellow/package looks good, yeah! ![image](https://user-images.githubusercontent.com/1244307/103038768-2fc01e80-4567-11eb-879a-bdc0af168ddc.png)

Reply:
Sure. 👍